preview-env/create/templates/comment-failure.md

```diff
@@ -2,3 +2,19 @@
 * __Status:__ :x:
 * __ArgoCD:__ :link: [Link](https://${ARGOCD_SERVER}/applications/argocd/${APP_NAME}?view=tree&resource=)([degraded resources](https://${ARGOCD_SERVER}/applications/argocd/${APP_NAME}?view=tree&resource=health%3ADegraded))
 * __Deployment Jobs:__ :clipboard: [Link](https://github.com/${REPO}/actions/runs/${RUN_ID})
+* __Troubleshooting Tips:__ :ring_buoy:
+```
+Typical Errors
+--------------
+ErrImagePull / ImagePullBackOff - The CI hasn't finished building the image yet -> increase "argocd_wait_for_sync_timeout"
+Readiness / Liveness probe failed - The container is not fully functional yet -> increase "argocd_wait_for_sync_timeout"
```

Comment:
> The container is not fully functional yet

Same case as above I think, the container may never become healthy due to a failure at startup so increasing the wait time will not help. People have to check container logs for the underlying problem before increasing the wait time.